jetty-hazelcast/src/main/java/org/eclipse/jetty/hazelcast/session/HazelcastSessionDataStore.java

```diff
@@ -184,7 +187,52 @@ public boolean isPassivating()
             }
             return false;
         } ).collect( Collectors.toSet() );
+        
+        
+        //Now find other sessions in hazelcast that have expired
+        final AtomicReference<Set<String>> reference = new AtomicReference<>();
+        final AtomicReference<Exception> exception = new AtomicReference<Exception>();
+        
+        Runnable r = new Runnable()
+        {
+            @Override
+            public void run ()
+            {
+                try
+                {
+                    Set<String> ids = new HashSet<>();
+                    EntryObject eo = new PredicateBuilder().getEntryObject();
+                    Predicate<?, ?> predicate = eo.get("expiry").greaterThan(0).and(eo.get("expiry").lessEqual(now));
```

Comment:
Good one. I wonder if we could use that somewhere else in the code (but yes part of an other change :) )